cmd/main.go

```diff
@@ -21,10 +22,13 @@ func main() {
 	if err != nil {
 		logger.Error(err)
 	}
-
 	dataStore := data.NewDataStore(database)
 	dataInteractor := interactor.NewDataInteractor(dataStore)
-	dataController := controllers.NewDataController(dataInteractor)
+
+	cloud := cloud.NewCloud(config.Cloud.Host, config.Cloud.Port)
+	cloudInteractor := interactor.NewCloudInteractor(cloud)
+
+	dataController := controllers.NewDataController(dataInteractor, cloudInteractor)
```

Comment:
As discussed with @joaoaneto, instead of creating another interactor (which should be associated with a use-case, not to mention other architectural reasons), the cloud should be used the the `dataInteractor` :grimacing: 

Change this (and apply all other necessary modifications for this to work) :point_down: 
```suggestion
	dataStore := data.NewDataStore(database)
	cloud := cloud.NewCloud(config.Cloud.Host, config.Cloud.Port)
  dataInteractor := interactor.NewDataInteractor(dataStore, cloud)
	dataController := controllers.NewDataController(dataInteractor)
```

Reply:
@joaoaneto done!